_assets/compose/mailserver/Makefile

```diff
@@ -0,0 +1,65 @@
+GIT_ROOT = $(shell git rev-parse --show-toplevel)
+
+RED    := $(shell tput -Txterm setaf 1)
+GREEN  := $(shell tput -Txterm setaf 2)
+YELLOW := $(shell tput -Txterm setaf 3)
+RESET  := $(shell tput -Txterm sgr0)
+BOLD   := $(shell tput bold)
+
+# Settings
+export LOG_LEVEL = INFO
+export LISTEN_PORT ?= 30303
+export CONTAINER_NAME ?= status-go-mailserver
+export DATA_PATH ?= /var/tmp/status-go-mail
+# Necessary to make mailserver available publicly
+export PUBLIC_IP ?= $(shell curl -s https://ipecho.net/plain)
+
+FLEETS_URL := https://fleets.status.im/
+FLEET_NAME := eth.beta
+FLEETS = $(shell curl -s "$(FLEETS_URL)")
+BOOT_NODES = $(shell echo '$(FLEETS)' | jq '.fleets["$(FLEET_NAME)"].boot | map(.)')
+
+all: checks start show info enode
+
+checks:
+ifeq (, $(shell which docker-compose))
+	$(error No $(BOLD)docker-compoase$(RESET) in your $$PATH. Please install it)
+endif
+ifeq (, $(shell which jq))
+	$(error No $(BOLD)jq$(RESET) in your $$PATH. Please install it)
+endif
+
+start: config
+ifndef PUBLIC_IP
+	$(error PUBLIC_IP not set! Export it as environment variable)
+endif
+ifndef CONTAINER_NAME
+	$(error CONTAINER_NAME not set! Export it as environment variable)
+endif
+	docker-compose up -d
+
+stop:
+	docker-compose down
+
+logs:
+	docker-compose logs -f -t --tail=100
+
+enode:
+	@docker-compose logs --tail=all | grep -oP '.*self="\Kenode://[^?]+' | sed 's/127.0.0.1/$(PUBLIC_IP)/'
```

Comment:
Another option might be to use IPC file and `admin.nodeInfo` method but it assumes IPC is created.

Reply:
Done.